app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -324,6 +324,13 @@ def idv_verification_submitted(
       )
     end
 
+    # Track when idv verification is rate limited during idv flow
+    def idv_verification_rate_limited
+      track_event(
+        :idv_verification_rate_limited,
+      )
+    end
```

Comment:
We've been trying to keep these in alphabetical order, so this should be moved up above `idv_verification_submitted`.